style/index.css

```diff
@@ -2,4 +2,5 @@
 @import url('https://fonts.googleapis.com/css2?family=Libre+Franklin:wght@700&display=swap');
 @import url('base.css');
 @import url('layout.css');
-@import url('modules/sectionHeading.css');
\ No newline at end of file
+@import url('modules/sectionHeading.css');
+@import url('modules/grid.css');
```

Comment:
FYI, this is why github warns about the missing newline character -- if line 5 had ended with one, we wouldn't be seeing it in the diff.

This is just a tooling thing, and it will be nice to solve it early on...

@Ernesto2004 @Tara-Tiong @kingtrentiii please head to Preferences > Settings, search for "newline," and set it like this:

<img width="976" alt="Screen Shot 2020-08-25 at 11 26 25 PM" src="https://user-images.githubusercontent.com/733916/91268611-812bde80-e72a-11ea-8594-2e6298d58bc9.png">